benchmarks/communication/utils.py

```diff
@@ -108,6 +108,11 @@ def get_bw(comm_op, size, duration, args):
     n = dist.get_world_size()
     tput = 0
     busbw = 0
+
+    if duration == 0:
+        print_rank_0("Error. duration is 0.")
```

Comment:
Nit: Typo, "duration" should be capitalized after period. Alternately replace the previous period with a ":".

Reply:
thanks! addressed it accordingly.